### DISCUSSION TITLE
How do I fetch a single issue and query specific fields on it (e.g. status)?

### DISCUSSION THREAD

**Discussion by erawhctim** [ankitpokhrel/jira-cli#786] [Q&A]
I assumed this was built into the CLI, but I can't seem to figure it out (I feel like I'm missing something obvious).

How can I look up the status of a particular ticket? I'm expecting something like `jira issue view {ticket ID} --status` or `jira issue view {ticket ID} --fields=status` or something.

**Comment by erawhctim** [comment]
Digging further, I think this is _close_ to what I want: 
`jira issue list -q"key = {ticket ID}" --columns status --plain --no-headers`

However, this still outputs the ticket ID as well: 
```
{ticket ID}	Close
```

**Is there any way to _just_ output the fields/columns I requested?** Ideally I would get back output like this: `Close` (assuming the status is "Close")

It'd be nice if this worked like the `--json` output flag works in the GH CLI: you can opt-in to raw/plain/JSON output, but also see the formatted output by default... 

(relates to #228)

**Reply by ValentinVoigt** [reply]
Just pass it through awk?

`jira issue list -q"key = {ticket ID}" --columns status --plain --no-headers | awk '{print $2}'`

**Reply by erawhctim** [reply]
that certainly works, yep
